packages/shorebird_cli/lib/src/shorebird_flavor_mixin.dart

```diff
@@ -5,6 +5,19 @@ import 'package:path/path.dart' as p;
 import 'package:platform/platform.dart';
 import 'package:shorebird_cli/src/shorebird_java_mixin.dart';
 
+class MissingGradleWrapperException implements Exception {
```

Comment:
A comment about why this would be thrown would be helpful to future Us

Reply:
good call, done ✅